### ISSUE TITLE
Cannot resolve module 'invariant'

### ISSUE THREAD

**Issue by sorenbs** [fullstackreact/react-yelp-clone#6]
Cloning the repo, running `npm i` and `npm start` gives me the following error when I load the page.

I am running on a mac.

I have tried removing the node_modules folder and installing again, but it didn't help. Any clues?

```
ERROR in ./~/google-maps-react/dist/index.js
Module not found: Error: Cannot resolve module 'invariant' in /Users/sorenbs/Code/react-yelp-clone/node_modules/google-maps-react/dist
 @ ./~/google-maps-react/dist/index.js 3:4-184

ERROR in ./~/google-maps-react/dist/lib/GoogleApi.js
Module not found: Error: Cannot resolve module 'invariant' in /Users/sorenbs/Code/react-yelp-clone/node_modules/google-maps-react/dist/lib
 @ ./~/google-maps-react/dist/lib/GoogleApi.js 3:8-49
```

**Comment by insin** [comment]
This is caused by fullstackreact/google-maps-react#3

**Comment by eonblue4309** [comment]
In the mean time, install invariant from npm as a dependency and you should be good to go.

**Comment by hugofqueiros** [comment]
+1 to fix this

**Comment by auser** [comment]
Fixed in google-maps-react#3.

Thanks for reporting!
